owncloudApp/src/main/res/values/strings.xml

```diff
@@ -56,11 +56,11 @@
     <string name="prefs_touches_with_other_visible_windows">Touches with other visible windows</string>
     <string name="prefs_touches_with_other_visible_windows_summary">Allow touches when the view is obscured by another visible window. Enable it to use light filtering apps.</string>
     <string name="confirmation_touches_with_other_windows_title">Are you sure you want to enable this feature?</string>
-    <string name="confirmation_touches_with_other_windows_message">Use this feature under your own responsibility, a malicious application could try to spoof you into performing some actions, unaware, using other views.</string>
-    <string name="prefs_subsection_picture_uploads">Auto upload pictures</string>
-    <string name="prefs_subsection_picture_uploads_summary">Manage location and behaviour of the pictures uploaded automatically</string>
-    <string name="prefs_subsection_video_uploads">Auto upload videos</string>
-    <string name="prefs_subsection_video_uploads_summary">Manage location and behaviour of the videos uploaded automatically</string>
+    <string name="confirmation_touches_with_other_windows_message">Use this feature at your own risk. A malicious application could try to spoof you into unknowingly performing some actions, using other views.</string>
+    <string name="prefs_subsection_picture_uploads">Auto-upload pictures</string>
+    <string name="prefs_subsection_picture_uploads_summary">Manage location and behavior of the pictures uploaded automatically</string>
```

Comment:
Well, I was taking a look at the code and for variable names and comments sometimes we use the UK variant ("behaviour") and sometimes the USA variant ("behavior"), so I guess both are valid

Reply:
Sure, both are valid, but this PR is part of an effort to improve the user experience by making the language more consistent/unified across all ownCloud products. Since we use US English in the documentation, it make sense to do so in the UI as well.